dll/win32/shell32/lang/he-IL.rc

```diff
@@ -341,15 +343,15 @@ FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
     GROUPBOX "בדיקת שגיאות", -1, 5, 5, 230, 60
     ICON IDI_SHELL_TOOLS_CHKDSK, IDC_STATIC, 13, 25, 21, 20
-    LTEXT "This option will check the volume for\nerrors.", -1, 40, 25, 160, 20
+    LTEXT "אפשרות זו תבדוק אם ישנן שגיאות בכונן.", -1, 40, 25, 160, 20
     PUSHBUTTON "בדוק כעת...", 14000, 130, 45, 90, 15, WS_TABSTOP
     GROUPBOX "איחוי", -1, 5, 65, 230, 60
     ICON IDI_SHELL_TOOLS_DEFRAG, IDC_STATIC, 13, 85, 21, 20
-    LTEXT "This option will defragment files on the volume", -1, 40, 85, 160, 20
+    LTEXT "אפשרות זו תאחה קבצים שבכונן.", -1, 40, 85, 160, 20
```

Comment:
Ditto.

Reply:
![image](https://user-images.githubusercontent.com/20986867/62944105-adf1aa00-bde4-11e9-8a58-ac861a57b7b0.png)

@binarymaster Appears Correctly In Notepad.
So what seems to be the issue?

Reply:
![image](https://user-images.githubusercontent.com/578406/62946294-1773b780-bde9-11e9-9e84-3abd1055858a.png)

Reply:
i'd like to apologize i don't think i quite understand. it appears correctly in notepad so shouldn't it be fine? isn't this just the Github preview that is broken?

Reply:
Ok I checked there are similar things already do exist in this file, and CI seems happy with it. Maybe this is indeed GitHub preview or the browser problem.

See https://patch-diff.githubusercontent.com/raw/reactos/reactos/pull/1828.patch - `IDI_SHELL_TOOLS_DEFRAG` and lines around it.

Reply:
That seem to make sense. i don't think Github is Designed to work with RTL.